### ISSUE TITLE
Increasing BCon.timeout attribute for Historical Index Composition

### ISSUE THREAD

**Issue by ahmasoh1992** [matthewgilbert/pdblp#34]
Hey!

Great work on the API - using it lots and it's doing a jolly good job. 

I'm running into an issue where when I use the following:

```python
X = con.ref('RAY Index','INDX_MWEIGHT_HIST', [('END_DT','19991231'),])
```

I end up facing the error:

```
raise RuntimeError("Timeout, increase BCon.timeout attribute")
RuntimeError: Timeout, increase BCon.timeout attribute
```

How can I manually hack the override in relation to this function call?

**Comment by matthewgilbert** [comment]
As of 2f8c3c82c743ac0c479fb3a914315242c066bcd2 you can set this in the constructor. Something like
```python
import pdblp
con = pdblp.Bcon(timeout=5000)
```

**Comment by matthewgilbert** [comment]
This is available for `version >= 0.1.3` on pypi as well.
